### PR TITLE
improve get_version_of_tools

### PR DESCRIPTION
**Pull request by TomasTomecek**
@twaugh PTAL

### CODE REVIEW

File: atomic_reactor/util.py [hunk 1]
```diff
@@ -459,18 +459,19 @@ def get_version_of_tools():
 
     :returns dict, {"tool": "version", ...}
```
Comment (twaugh) [inline atomic_reactor/util.py]: Comment needs updating.

**Comment by twaugh** [comment]
Other than the docstring it looks great!

**Comment by TomasTomecek** [comment]
Good catch, fixed, thanks!

**Comment by twaugh** [comment]
LGTM

**Pull request merged**